public/locales/en/settings.json

```
@@ -3,6 +3,16 @@
   "save": "Save",
   "saving": "Saving...",
   "reset": "Reset",
+  "language": "Language",
```

Would it be helpful to clarify what is the target of language picker? 
For example:
```suggestion
  "language": "Web UI Language",
```

---

I don't think it's necessary. It's expected to change the Web UI language, as we are in the Web UI settings no? 🤔

---

I think `Language` is ok.